components/bpmn/bpmn-rest/src/main/java/org/wso2/carbon/bpmn/rest/service/stats/UserService.java

```diff
@@ -80,20 +83,20 @@ public ResponseHolder getNoOfTasksCompletedByUser() throws UserStoreException {
         try {
             String[] users = (String[]) getUserList().getData().toArray();
 
-        for (String u : users) {
-            UserTaskCount userInfo = new UserTaskCount();
-            userInfo.setUserName(u);
+            for (String u : users) {
+                UserTaskCount userInfo = new UserTaskCount();
+                userInfo.setUserName(u);
 
-            long count = BPMNOSGIService.getHistoryService()
-                    .createHistoricTaskInstanceQuery().taskAssignee(u).finished().count();
-            if (count == 0) {
-                userInfo.setTaskCount(0);
-            } else {
-                userInfo.setTaskCount(count);
+                long count = BPMNOSGIService.getHistoryService()
+                        .createHistoricTaskInstanceQuery().taskTenantId(str).taskAssignee(u).finished().count();
+                if (count == 0) {
```

Comment:
This if else is not required. you can directly use userInfo.setTaskCount(count);